tools/build_langserver/langserver/utils.go

```diff
@@ -65,6 +65,30 @@ func GetPathFromURL(uri lsp.DocumentURI, pathType string) (documentPath string,
 	return "", fmt.Errorf(fmt.Sprintf("invalid path %s, path must be in repo root: %s", absPath, core.RepoRoot))
 }
 
+// LocalFilesFromURI returns a slices of file path of the files in current directory
+// where the document is
+func LocalFilesFromURI(uri lsp.DocumentURI) ([]string, error) {
+	fp, err := GetPathFromURL(uri, "file")
+	if err != nil {
+		return nil, err
+	}
+
+	var files []string
+
+	fpDir := filepath.Dir(fp)
+	err = fs.Walk(fpDir, func(name string, isDir bool) error {
```

Comment:
You probably don't need a recursive `Walk` here - probably `ioutil.ListDir` is enough.

Reply:
Oh! let me give that a go!